src/Microsoft.SqlTools.ServiceLayer/Profiler/ProfilerSessionMonitor.cs

```diff
@@ -32,16 +32,38 @@ public class ProfilerSessionMonitor : IProfilerSessionMonitor
 
         private Task processorThread = null;
 
-        private Dictionary<string, ProfilerSession> monitoredSessions = new Dictionary<string, ProfilerSession>();
+        public struct Viewer
```

Comment:
if this class is internal implementation detail you may consider making it `private` or `internal`